alcs-frontend/src/app/features/notice-of-intent/applicant-info/notice-of-intent-details/roso-details/roso-additional-information/roso-additional-information.component.html

```diff
@@ -0,0 +1,73 @@
+<div *ngIf="_noiSubmission" class="review-table">
```

Comment:
Can probably rename to additiona-information-component since the 3 of them are nearly identical.

Reply:
I'll do renaming in scope of another ticket that adds other types